myapp/src/App.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import { Toy } from './components/Toy' 
+
+export default class App extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      toys: [
```

Comment:
e indicat ca in constructor sa initializezi state-ul cu valori default, apoi in componentDidMount poti modifica acest state, cu date concrete (venite de pe un server, props etc)

ex: 
this.state = { toys: [] }